src/reachy2_sdk/parts/head.py

```diff
@@ -109,9 +127,24 @@ def get_joints_positions(self) -> List[float]:
     def look_at(
         self, x: float, y: float, z: float, duration: float = 2.0, wait: bool = False, interpolation_mode: str = "minimum_jerk"
     ) -> GoToId:
-        """Compute and send neck rpy position to look at the (x, y, z) point in Reachy cartesian space (torso frame).
+        """Compute and send a neck position to look at a specified point in Reachy's Cartesian space (torso frame).
+
+        The (x, y, z) coordinates are expressed in meters, where x is forward, y is left, and z is upward.
+
+        Args:
+            x: The x-coordinate of the target point.
+            y: The y-coordinate of the target point.
+            z: The z-coordinate of the target point.
+            duration: The time in seconds for the neck to reach the target point. Defaults to 2.0.
```

Comment:
I would say "for the neck to look at the point" more than 'reach the point' to be clearer